src/hangar/config_hangar.yml

```diff
@@ -5,7 +5,7 @@ hangar:
   __version__: '0.0.0'
 
   lmdb:
-    map_size: 10_000_000_000
+    map_size: 50_000_000
```

Comment:
So by reducing this to 50MB, we were able to get the windows test to start working (and failing for real reasons, not "out of disk space") apparently there's a large difference in how mmapped files are dealt with between unix and windows...

This needs to be reverted back before we merge. but just leaving it as a blocking issue here until we find a solution.